geode-redis/README.md

```diff
@@ -1,78 +1,302 @@
-The `benchmark.sh` script uses several helper scripts. The `execute-operation.sh` script runs a
-particular instance of `redis-benchmark`. The `summarize-operation-results.sh` script processes the
-output of `execute-operation.sh`, and the `summarize-batch-results.sh` script processes the output
-of multiple runs of `summarize-operation-results.sh`.
+5. Once in that folder run the following command:
+    ```commandline
+   $ ./gfsh
+   ```
 
-###The `environment-setup.sh` Script
-The `environment-setup.sh` is optional. It can start a local Geode Redis Adapter, or confirm that a
-local Redis server is running, then call `benchmark.sh`.
+You should now see GFSH starting up with a version of 1.14.x.-build.x
 
-Mandatory command line arguments for `environment-setup.sh`:
-- either `-g` or `-r`
+![screenshot of GFSH running in the terminal](gfsh.png)
 
-`-g` will:
-- Start a local Geode Redis Server for you
-- Shut the server down once the benchmark finishes
+### <a name="starting-a-server"></a>Starting a Geode Server with Redis Enabled
+Using GFSH enter the following commands:
 
-`-r` will:
-- Connect to a Redis server that is already running
+1. Start a locator. The locator tracks servers and server load. When a client requests a server 
+connection, the locator directs the client to one of the least loaded servers. Learn more. 
+   ```commandline
+    gfsh> start locator
+    ``` 
 
-Optional command line arguments for `environment-setup.sh`:
-- `-f` indicates an optional prefix for the summary file name that will be passed to `benchmark.sh`
+2. After the locator has started, start a server that will be able to handle incoming Redis commands. 
 
-###The `shacompare.sh` Script
-The `shacompare.sh` script is used to compare performance between different commits. This script
-takes in two different commit hashes, checks them out, uses `environment-setup.sh` script to start
-up the Geode Redis server, and runs the benchmarks for each commit. It generates two summary CSV
-files, labeled with the short commit hash, that can be compared for performance changes.
+    For example:
+    ```commandline
+    gfsh> start server --name=redisServer1 --locators=localhost[10334] --server-port=0 --redis-port=6379 --redis-bind-address=127.0.0.1
+    ```
+    * --name: A name you create for your server.
+    * --locators: This is the location of the locator you started in step 1. 
+    * --server-port: The port that Geode clients connect to.
+    * --redis-port: The port that your Redis client will connect to.
+    * --redis-bind-address: This will be the address your client uses to connect. 
+
+    Your Geode instance should now be up and running (1 locator and 1 server) and ready to accept Redis 
+    commands.  
+
+    **Keep this terminal open and running so that you can easily shutdown the Geode instance when you are 
+    done working locally.**
+
+3. To confirm that things are running correctly, in a separate terminal run:
+      ```commandline
```

Comment:
Should we say something here about the version of redis they should use and how to get the redis-cli installed?

Reply:
I added a subtle version mention to the intro paragraph, let me know what you think!